templates/components/user/create_user.html.twig

```diff
@@ -0,0 +1,43 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2022 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+<button type="button"
```

Comment:
Current design is not ideal i think but i don't have ideas about it currently.

Some suggestions despite that:
- move to `ti ti-user-plus` icon
- a single button for all actors could be better no ?
- do we have any way to disable the display of the button

Reply:
maybe something like that if there is no conflict event with accordion

![image](https://user-images.githubusercontent.com/418844/166638408-22259e7f-a9f4-4cc3-abf8-69eb96ff6eb8.png)

Reply:
> maybe something like that if there is no conflict event with accordion

:+1: I prefer this design.

Reply:
Button moved, but I can't manage not to trigger the folding/unfolding of the menu.

Reply:
> I can't manage not to trigger the folding/unfolding of the menu.

I fixed it.

Reply:
> I fixed it.

Thank you